integration/echo_server/clmigration.js

```diff
@@ -0,0 +1,10 @@
+// clmigration prepares plumbing for correct async/await behavior, in spite of
```

Comment:
I think we can get rid of the migrations all together in the examples, in which case we wouldn't need this file.